torch/_inductor/fx_passes/decompose_mem_bound_mm.py

```diff
@@ -17,16 +17,30 @@
 # TODO: need a better strategy for decomposing mm
 MIN_FIRST_DIMENSION_DECOMPOSITION = 10240
 MAX_OTHER_DIMENSION_DECOMPOSITION = 32
+CPU_MAX_FIRST_DIMENSION_DECOMPOSITION = 1
+CPU_MAX_OTHER_DIMENSION_DECOMPOSITION = 2048
 
 min_first_dimension_decomposition = MIN_FIRST_DIMENSION_DECOMPOSITION
 max_other_dimension_decomposition = MAX_OTHER_DIMENSION_DECOMPOSITION
+cpu_max_first_dimension_decomposition = CPU_MAX_FIRST_DIMENSION_DECOMPOSITION
```

Comment:
curious if we can reuse MIN_FIRST_DIMENSION_DECOMPOSITION?

Reply:
I would prefer to create a new one for the CPU case as the existing MIN_FIRST_DIMENSION_DECOMPOSITION is mostly used for CUDA case, and the number could be changed/diverged in the future.